site/docs/remote-caching.md

```diff
@@ -306,6 +307,31 @@ You may want to delete content from the cache to:
 * Create a clean cache after a cache was poisoned
 * Reduce the amount of storage used by deleting old outputs
 
+## Build cache
+
+Bazel can use a directory on the filesystem as a remote cache. This is
+useful for sharing build artifacts when switching branches and/or working
+on multiple workspaces of the same project i.e. multiple checkouts. Bazel
+does not garbage collect the directory and thus one might want to set up a
+cron job or similar to periodically clean up the directory. The build cache
+can be enabled with the following flags.
+
+```
+build --build_cache=/path/to/build/cache
+```
+
+Additionally, one can pass a user specific path to the --build_cache flag
+via the ~ character. Bazel will replace this with the current user's home
+directory. This comes in handy when one wants to enable the build cache for
+all developers of a project via the project's checked in `.bazelrc` file.
+
+To have cache hits across different workspaces, additional option should be
+used:
```

Comment:
I removed the the listing of the env. variables. The use case I'm after is: the same user (no multiple uses involved) has different clones of the same project, say:

* /home/davido/projects/gerrit
* /home/davido/projects/gerrit2

After building in one directory, the cache is populated. When I build in another directory, without `--experimental_strict_action_env` I experienced cache misses. I havn't rechecked whether this is still the case, but @ulfjack recommended to use this option in this comment: https://github.com/bazelbuild/bazel/issues/3042#issuecomment-330151933 and pointed to the discussion in context of this issue: https://github.com/bazelbuild/bazel/issues/2574